library/src/main/res/values/library_mpandroidchart_strings

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+	<string name="define_mpandroidchart"></string>
+	<!-- Author section -->
+	<string name="library_mpandroidchart_author">PhilJay</string>
+	<string name="library_mpandroidchart_authorWebsite">https://github.com/PhilJay/MPAndroidChart</string>
+	<!-- Library section -->
+	<string name="library_mpandroidchart_libraryName">MPAndroidChart</string>
+	<string name="library_mpandroidchart_libraryDescription">MPAndroidChart is a powerful and easy to use chart library for Android, supporting line-, bar-, scatter-, candlestick-, bubble-, pie- and radarcharts (spider web), as well as scaling, dragging (panning), selecting and animations.</string>
+	<string name="library_mpandroidchart_libraryWebsite">https://github.com/PhilJay/MPAndroidChart</string>
+	<string name="library_mpandroidchart_libraryVersion">v2.0.8</string>
+	<!-- OpenSource section -->
+	<string name="library_mpandroidchart_isOpenSource">true</string>
+	<string name="library_mpandroidchart_repositoryLink">https://github.com/PhilJay/MPAndroidChart</string>
+	<!-- ClassPath for autoDetect section -->
+	<string name="library_mpandroidchart_classPath"></string>
```

Comment:
You should also define a classPath. This is a class (the whole packagename inclusive class name) which is one of the main classes inside the library. So the AboutLibraries library can autodetect this one.